src/helm.exec.ts

```diff
@@ -297,7 +298,7 @@ export async function helmListAll(namespace?: string): Promise<Errorable<string[
         return { succeeded: false, error: [ "Helm client is not installed" ] };
     }
 
-    const releases: string[] = [];
+    const releases = [];
```

Comment:
I suggest leaving the type annotation in: when I switch on `strictNullChecks`, the compiler will start inferring the naked literal as type `never[]` and will complain.  Even without that, declaring the type at this point ensures that adding the wrong kind of thing to it will typecheck.